Yosemite/Yosemite/Model/Mocks/Graphs/ScreenshotsObjectGraph.swift

```diff
@@ -215,7 +215,7 @@ struct ScreenshotObjectGraph: MockObjectGraph {
             granularity: .yearly,
             intervals: thisYearVisitStats.items!.enumerated().map {
                 Self.createInterval(
-                    monthsAgo: $0.offset,
+                    date: Date().yearStart.addingMonths($0.offset),
```

Comment:
Yup, updated here as well in https://github.com/woocommerce/woocommerce-ios/pull/5936/commits/19333241c80dd2f9e0159c51156b350b6844ffea so that both places share the same date